setup.py

```diff
@@ -46,6 +46,7 @@
         'jinja2 >= 2.10',
         'protobuf >= 3.7.1',
         'pypandoc >= 1.4',
+        'PyYAML >= 5.1.1',
```

Comment:
Do we need the PyYAML inclusion here right now? AFAIK, the only place where we use PyYAML is in the tests.

I assume when you do read an actual external sample config YAML file, you will need it.

Reply:
Right now, PyYAML is only needed for the test because the code to read in the sample config isn't hooked up. I suppose this then counts as a spurious dependency.